datastore/package.json

```diff
@@ -4,6 +4,9 @@
   "version": "0.0.1",
   "private": true,
   "license": "Apache Version 2.0",
+  "bin": {
```

Comment:
Yeah, probably not worth dealing with issues about the symlink failing due to permissions and such. It does allow a nicer interface, though:

```sh
$ datastore-tasks list
```

vs.

```sh
$ npm run datastore-tasks list
```

I'll switch this over to the second style.

Regarding the consistency note, do you prefer `datastoreTasks` to match `authSample`? Better yet, maybe just `tasks`?:

```sh
$ npm run tasks list
```